src/program.js

```diff
@@ -184,6 +184,10 @@ Example: $0 --help run.
         demand: false,
         type: 'string',
       },
+      'no-reload': {
```

Comment:
@kumar303 how about defaulting `noReload` to `true` until we handle the issues with the XPI file caching ([Bug 1268884](https://bugzilla.mozilla.org/show_bug.cgi?id=1268884) and `SIGINT` support on windows (webext issue #225)?